Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class Lesson4
+    {
+        public static void Demo()
+        {
+            PromptPersonData();
+            PromptPersonData();
+        }
+        private static void PromptPersonData()
+        {
+            string firstName = NamePrompt("What is your first name: ");
+            string lastName = NamePrompt("What is your last name: ");
+            int age = PromptInt("What is your age: ");
+            float weight = PromptFloat("what is your weight in Kg: ");
+            float height = PromptFloat("what is your height in cm: ");
+
+            Console.WriteLine($"{firstName} {lastName} is {age} years old, his weight is {weight} kg and his height is {height} cm. ");
+
+            float bmi = BmiCalculator(weight, height / 100);
+            Console.WriteLine($"Your BMI is {bmi}");
+        }
+        public static string NamePrompt(string message)
+        {
+            string name = PromptString(message);
+
+            if (name.Equals("-"))
+            {
+                Console.WriteLine("Name cannot be empty.");
+            }
+
+            return name;
+        }
+        public static string PromptString(string message)
+        {
+            Console.Write(message);
```

Comment:
I think this has to be WriteLine instead.

Reply:
It's okay, spec was poor.